docs/tutorials/hobby_calibration.rst

```diff
@@ -7,9 +7,11 @@
 Calibrating Modules With Analog Hobby Protocols
 ***********************************************
 
-Vertiq modules can be controlled with analog hobby protocols, such as PWM and OneShot. These protocols send throttle commands to the module using a square wave or pulse of variable duration,
-where the length of the wave determines the throttle input. For example, when using PWM, by default Vertiq modules interpret a pulse of 1000 us as 0% throttle, and a pulse of 2000 us
-as 100% throttle.
+Vertiq modules can be controlled with analog :ref:`hobby protocols <manual_hobby>`, such as :ref:`Standard PWM <hobby_standard_pwm>` and :ref:`OneShot <hobby_other_protocols>`. 
```

Comment:
Using Standard PWM instead of just saying PWM to be consistent with other documents